packages/pds/tests/sync/subscribe-repos.test.ts

```diff
@@ -323,6 +372,64 @@ describe('repo subscribe repos', () => {
     verifyHandleEvent(handleEvts[0], bob, 'bob2.test')
   })
 
+  it('syncs account events', async () => {
+    // deactivate then reactivate alice
+    await agent.api.com.atproto.server.deactivateAccount(
+      {},
+      {
+        encoding: 'application/json',
+        headers: sc.getHeaders(alice),
+      },
+    )
+    await agent.api.com.atproto.server.activateAccount(undefined, {
+      headers: sc.getHeaders(alice),
+    })
+
+    // takedown then restore bob
```

Comment:
I _think_ that it's covered alright, but what about flows where alice is deactivated while takendown, or is takendown while deactivated?  (Or activated while takendown?  I think I see this one at least would error.)

Would they look right both to a firehose consumer and on `getRepoStatus`?

Reply:
good call, probably worth a test 👍